core/templates/pages/learner-dashboard-page/my-certificates-tab.component.html

```
@@ -0,0 +1,2 @@
+<!-- TODO(#24717-M2.12): This is a skeleton component created to establish the flow so that other PRs can be developed in parallel. The UI will be implemented in a future PR. -->
+<h1>My Certificates</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)